uploadNeuroDB/minc_insertion.pl

```diff
@@ -455,6 +464,40 @@
     NeuroDB::MRI::make_nii(\$file, $data_dir);
 }
 
+############################################################
+############# Create minc-pics #############################
+############################################################
+unless ($create_minc_pics)
+{
```

Comment:
@gluneau,

you want to create pics if the user spoecifies this option; this will do the opposite
(I just confirmed when running the command line: pics generated withOUT specifying -create_minc_pics)